client/a8c-for-agencies/sections/signup/primary/agency-signup/index.tsx

```diff
@@ -36,7 +38,7 @@ export default function AgencySignup() {
 	return (
 		<>
 			{ showSignupIntro && <SignupIntro wpAdminUrl={ wpAdminUrl } /> }
-			<SignupForm />
+			{ isEnabled( 'a4a-signup-v2' ) ? <SignupFormV2 /> : <SignupForm /> }
```

Comment:
We now render the new form in the component level instead of the controller to inherit some of the redirect logics.